lagunita/lms/templates/email_change_successful.html

```diff
@@ -0,0 +1,18 @@
+<%inherit file="main.html" />
+<%!
+from django.urls import reverse
+from django.utils.translation import ugettext as _
+%>
+<section class="container activation">
+  <section class="message">
+    <h1 class="valid">${_("E-mail change successful!")}</h1>
+    <hr class="horizontal-divider">
+    <p>${_("Your updated email address is now {new_email}.").format(
+        new_email=new_email,
+    )}</p>
+    <p>${_("To view your courses, go to your {link_start}dashboard{link_end}.").format(
+        link_start='<a href="{url}">'.format(url=reverse('dashboard')),
+        link_end='</a>',
+      )}</p>
```

Comment:
Indentation isn't consistent with first paragraph element?

Reply:
It looks like that's how that line was formatted upstream in platform.

I didn't change it, but could.

Reply:
Yea, since we're overriding anyways so may as well be consistent I feel. It seems like they don't really stick with any standard indentation / formatting guidelines with the platform templates...

Reply:
@kluo Pushed per f2f.